Mathlib/Algebra/Star/Unitary.lean

```diff
@@ -58,6 +59,10 @@ theorem star_mul_self_of_mem {U : R} (hU : U ∈ unitary R) : star U * U = 1 :=
 theorem mul_star_self_of_mem {U : R} (hU : U ∈ unitary R) : U * star U = 1 :=
   hU.2
 
+@[simp]
+lemma one_mem_unitary : 1 ∈ unitary R :=
+  one_mem _
+
```

Comment:
This feels a bit weird; should we just tag `one_mem` with simp? Or if that keys badly, `Submonoid.one_mem`?

Reply:
```suggestion
```